kits/mou-drafter/apps/public/swiftlatex/pdftex_tikz.html

```
@@ -0,0 +1,287 @@
+    		max-height: 400px;
+    		overflow-y: scroll;
+		}
+    </style>
+</head>
+<body class="code-snippets-visible">
+
+    <div class="docs-section" id="demo">
+        <div id="editor">% Random city
+% Author: Pascal Seppecher
+\documentclass[tikz,border=10pt]{standalone}
+%%%<
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)